courses-app/src/services/CoursesService.jsx

```diff
@@ -0,0 +1,24 @@
+import axios from 'axios';
+
+export const getCourses = () => axios.get('http://localhost:4000/courses/all');
+export const getAuthors = () => axios.get('http://localhost:4000/authors/all');
+
+export const createAuthor = ((author) => {
+  console.log(author);
+  const config = {
+    headers: {
+      'Content-type': 'application/json',
+      Authorization: localStorage.getItem('token'),
+    },
+  };
+  axios.post('http://localhost:4000/authors/add', author, config);
+});
+export const createCourse = ((course) => {
+  const config = {
+    headers: {
+      'Content-type': 'application/json',
+      Authorization: localStorage.getItem('token'),
+    },
+  };
+  axios.post('http://localhost:4000/courses/add', course, config);
+});
```

Comment:
ovde imamo situaciju da za svaki request moramo pisati ovaj config, to nije dobro jer imamo ponavljanje koda.
ovo se moze bolje napisati. procitaj npr: https://stackoverflow.com/questions/40988238/sending-the-bearer-token-with-axios . postoje opcije da se pravi i custom hookovi i jos neke metode.